draft-wussler-openpgp-pqc.md

```diff
@@ -164,6 +168,8 @@ informative:
         ins: K. Randall
         name: Karen Randall
     date: October 2019
+    seriesinfo:
+      NIST Special Publication SP800-186
```

Comment:
```suggestion
      NIST Special Publication 800-186
```

Reply:
"SP" not needed if spelled out.

Aligned to the above proposal for the other reference SP800-56A.